_data/developer.yml

```diff
@@ -33,8 +33,9 @@ websites:
       url: https://bitbucket.org
       twitter: bitbucket
```

Comment:
Could you just removed the twitter handle as that's only used when the site doesn't support 2FA.